nixos/doc/manual/from_md/release-notes/rl-2211.section.xml

```diff
@@ -806,6 +806,23 @@
           for <literal>termonad</literal> has been removed.
         </para>
       </listitem>
+      <listitem>
+        <para>
+          The NixOS option <literal>nix.settings.max-jobs</literal> has
+          been changed from auto to 1. This default is the same as
+          documented in manpage for nix.conf and it prevents
+          administrator from high loads.
```

Comment:
English is not my 1st language, but that part sound a bit odd to me. How about `[...] prevents accidentally overloading the system.`?

Reply:
I removed the second part of the sentence.